content/docs/integrations/azure-devops/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Azure Devops
+publishedDate: '2022-04-06T11:00:00.0Z'
+description: Set up the Azure Devops plugin.
+
+humanName: Azure Devops
+logoImage: '../../../assets/logos/azure-devops/Azure-DevOps-logo.png'
+integrationType: OSS plugin
+---
+
+## Introduction
+
+[Microsoft’s Azure DevOps](https://dev.azure.com/) is a hosted service providing development and collaboration tools.
+
+This plugin provides visualisations for some components of Azure Devops in your components using annotations. 
+
+### Azure Pipelines
+Azure Pipelines is a CI/CD, testing, and deployment system that can connect to any Git repository. 
+
+The plugin lists the latest builds for a given Azure Repo.
+
+![Azure Piplines Builds screenshot](../../../assets/backstage/plugins/azure-devops/azure-devops-builds.png)
```

Comment:
Let's credit this image please. As far as I can tell it should be credited to [Andrew Wanlin from Keyloop](https://github.com/awanlin) who created this plugin.